infra/modules/service/execution-access.tf

```
@@ -0,0 +1,31 @@
+#-----------------
+# ECS Exec Access
+# See https://docs.aws.amazon.com/AmazonECS/latest/developerguide/ecs-exec.html
+#-----------------
+resource "aws_iam_policy" "ecs_exec" {
+  name        = "${var.service_name}-ecs-exec"
+  description = "A policy to run ECS Exec"
```

nit: for consistency our policy descriptions follow a grammar pattern of "Allows ..." e.g. [evidently policy](https://github.com/navapbc/platform-test/blob/86290ce79ea832c4313d18ad32d817dee686bcf7/infra/modules/feature-flags/access-policy.tf) (i realize there's some tech debt since not all of our policies have descriptions)

also, looking through the actual permissions in here, it looks like a more apt description is buried in a comment and could benefit from being brought into the description itself

```suggestion
  description = "Allow access to SSM Messages to support ECS Exec"
```